.github/workflows/dev-docker-publish.yml

```diff
@@ -31,20 +31,20 @@ jobs:
       - name: Deploy
         uses: WyriHaximus/github-action-helm3@v2
         with:
-          exec:
-            helm upgrade --namespace dev-inner-circle --create-namespace --install dev-salary-service-api
-            --set "aspnetcoreEnvironment=Development"
-            --set "ingress.host"=${{ secrets.DEV_HOST }}
-            --set "deployment.appsettings.ConnectionStrings.DefaultConnection=${{ secrets.DEV_POSTGRESQL_CONNECTION_STRING }}"
-            --set "deployment.appsettings.AuthenticationOptions.PublicSigningKey=${{ secrets.DEV_PUBLIC_SIGNING_KEY }}"
-            --set "deployment.appsettings.AuthenticationOptions.PrivateSigningKey=${{ secrets.DEV_PRIVATE_SIGNING_KEY }}"
-            --set "resources.limits.cpu=${{ secrets.DEV_APP_CPU_LIMIT }}"
-            --set "resources.limits.memory=${{ secrets.DEV_APP_MEMORY_LIMIT }}"
-            --set "resources.requests.cpu=${{ secrets.DEV_APP_CPU_REQUESTS }}"
-            --set "resources.requests.memory=${{ secrets.DEV_APP_MEMORY_REQUESTS }}"
-            --set "imageCredentials.username=${{ github.actor }}"
-            --set "imageCredentials.password=${{ secrets.GH_REGISTRY_TOKEN }}"
-            --set "image.tag=${{ github.sha }}"
-            --set "image.pullPolicy=Always"
-            ./SalaryService.Api/ci/chart
+          exec: |
+            RELEASE_NAME=dev-inner-circle-salary-service-api
+            helm repo add bitnami https://charts.bitnami.com/bitnami
+            helm upgrade --install --namespace dev-inner-circle --create-namespace --values ./SalaryService.Api/ci/values-custom.yaml \
+            --set "image.pullSecrets={${RELEASE_NAME}-aspnet-core-docker-secret}" \
+            --set "image.pullCredentials.username=${{ github.actor }}" \
```

Comment:
It seem that it works wrong for now. You need to create username and token for registry access in the ci secrets or use something like [this](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) (**Note:** but it only work for build, not for deploy)